src/layouts/ComponentLayout.astro

```diff
@@ -62,7 +79,7 @@ const examples = getAllExample(frontmatter.exampleKey);
         frontmatter.baseElement && (
           <p class="baseElement">
             <!-- eslint-disable-next-line -->
-            <code>&lt;{frontmatter.baseElement}&gt;</code>要素をベースにしています。 表にない属性もprop経由で使用できます（<code>className</code>は除く）。
+            <code>&lt;{frontmatter.baseElement}&gt;</code>要素をベースにしています。 表にない属性もprop経由で使用できます{frontmatter.allowClassName ? '' : '（classNameは除く）'}。
```

Comment:
If the element's attributes are available, they are flagged and this description is issued. However, since there are cases where className can be used and cases where it cannot be used, the description is now separated by a flag.